src/main/java/org/kainos/ea/resources/DeliveryEmployeeController.java

```diff
@@ -0,0 +1,37 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.cli.DeliveryEmployee;
+import org.kainos.ea.cli.DeliveryEmployeeRequest;
+import org.kainos.ea.cli.Employee;
+import org.kainos.ea.client.FailedtoCreateEmployee;
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+@Api("Evian Week 2 Exercise Delivery Employees API")
+@Path("/api")
+public class DeliveryEmployeeController {
+
+    private DeliveryEmployeeService deliveryEmployeeService = new DeliveryEmployeeService();
+    @GET
+    @Path("/delivery_employees")
```

Comment:
Would using a `-` character be a better choice instead of `_`? 

Could the `employee` and `delivery employee` api's be merged in any way?